### PR TITLE
Support text in stage channels

### PR DESCRIPTION
**Pull request by novanai**
### Summary

Support text in stage channels

### Checklist

- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

File: tests/hikari/impl/test_entity_factory.py [hunk 1]
```diff
@@ -2034,6 +2036,7 @@ def test_deserialize_guild_stage_channel_with_null_fields(self, entity_factory_i
                 "user_limit": 3,
                 "rtc_region": None,
                 "type": 6,
+                "last_message_id": None,
```
Comment (FasterSpeeding) [inline tests/hikari/impl/test_entity_factory.py:2039]: Should prob add `assert channel.last_message_id is None` to this test case and `test_deserialize_guild_stage_channel_with_unset_fields`

Comment (novanai) [reply tests/hikari/impl/test_entity_factory.py:2039]: done :)

File: hikari/impl/entity_factory.py [hunk 2]
```diff
@@ -1173,6 +1173,9 @@ def deserialize_guild_stage_channel(
             snowflakes.Snowflake(overwrite["id"]): self.deserialize_permission_overwrite(overwrite)
             for overwrite in payload["permission_overwrites"]
         }
+        last_message_id = (
+            snowflakes.Snowflake(msg_id) if (msg_id := payload.get("last_message_id")) is not None else None
+        )
```
Comment (davfsa) [inline hikari/impl/entity_factory.py:1178]:
```suggestion
        last_message_id = snowflakes.Snowflake(payload["last_message_id"]) if "last_message_id" in payload else None
```

Should make this a nicer one-liner

Comment (novanai) [reply hikari/impl/entity_factory.py:1178]: It was at first, I dunno if black was too happy with it though

Comment (davfsa) [reply hikari/impl/entity_factory.py:1178]: Line length is bellow 120, so shouldn't be too many issues. Regardless, reads nicer IMO

Comment (novanai) [reply hikari/impl/entity_factory.py:1178]: Okay :)

**Review by davfsa — APPROVED** [review]

**Pull request merged**